### DISCUSSION TITLE
Problem installing Elasticsearch-8.11.1 with Snowstorm-10.4.2.jar

### DISCUSSION THREAD

**Discussion by FilipTNH** [IHTSDO/snowstorm#623] [Q&A]
Elasticsearch 8.11.1(with built-in java environment) installs and launches fine (with xpack.security.enabled: false) after increasing vm.max_map_count as requested in log at startup.
Running Snowstorm.jar (using Java 17.0.12) gives a series of warnings on BeanPostProcessor + warning 'cannot find template location: classpath:/templates/'  + info 'error parsing HTTP request header'.  Starting RF2 SNAPSHOT import on branch MAIN runs as expected for a while but then Snowstorm crashes with no clear indication in the log file.  After restarting Snowstorm, you find a blocked MAIN branche with partial upload.
Any idea what might go wrong and how this can be fixed?

FYI - Installing ES7.10.2 and Snowstorm 9.2.0 on another machine and uploading INT goes smoothly.

**Comment by kaicode** [comment]
@FilipTNH is this still a problem? From your description it sounds likely that the semantic index processing was happening when Snowstorm was restarted. The logs go quiet for some time during this process while all relationships and axioms are read an processed.

If this is still a problem please create an issue in the [Snowstorm project issues page](https://github.com/IHTSDO/snowstorm/issues). Please include a copy of the Snowstorm and Elasticsearch logs from the time of the issue.

Kind regards,
Kai Kewley
